internal/querycoordv2/job/job_load.go

```diff
@@ -174,6 +158,31 @@ func (job *LoadCollectionJob) Execute() error {
 		job.undo.NewReplicaCreated = true
 	}
 
+	// 3. loadPartitions on QueryNodes
+	schema, err := job.broker.GetCollectionSchema(job.ctx, req.GetCollectionID())
+	if err != nil {
+		return err
```

Comment:
wrap it

Reply:
The `return err` below should also be wrapped too.

Reply:
done